core/helpers/EEH_Activation.helper.php

```diff
@@ -1079,7 +1095,7 @@ public static function initialize_system_questions()
                     );
                     $QST_ID = $wpdb->insert_id;
                     // QUESTION GROUP QUESTIONS
-                    if (in_array($QST_system, array('fname', 'lname', 'email'))) {
+                    if (in_array($QST_system, array('fname', 'lname', 'email', 'emailc'))) {
```

Comment:
I think we were planning on making this something the site admin would opt-in to by assigning the question to the question group.